charts/rancher-gke-operator/1.0.000/values.yaml

```diff
@@ -0,0 +1,11 @@
+global:
+  systemDefaultRegistry: ""
+
+gkeOperator:
+  image:
+    repository: rancher/gke-operator
+    tag: v1.0.0
```

Comment:
Bump this to 1.0.1 to match chart.

Reply:
@cbron we'll probably have rc images first, also why start at 1.0.1?

Reply:
Because of the leading zero's being invalid semver. Not supposed to do 000. 
So this tag can match the appVersion in Chart.yaml, and we would just cut rc's on that version until we are done.